iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/StaticVarCompensatorAdderImpl.java

```diff
@@ -59,17 +59,27 @@ public StaticVarCompensatorAdderImpl setBmax(double bMax) {
     }
 
     @Override
-    public StaticVarCompensatorAdderImpl setVoltageSetPoint(double voltageSetPoint) {
-        this.voltageSetPoint = voltageSetPoint;
+    public StaticVarCompensatorAdderImpl setVoltageSetpoint(double voltageSetpoint) {
+        this.voltageSetpoint = voltageSetpoint;
         return this;
     }
 
     @Override
-    public StaticVarCompensatorAdderImpl setReactivePowerSetPoint(double reactivePowerSetPoint) {
-        this.reactivePowerSetPoint = reactivePowerSetPoint;
+    public StaticVarCompensatorAdder setVoltageSetPoint(double voltageSetPoint) {
+        return setVoltageSetpoint(voltageSetPoint);
+    }
```

Comment:
If you add a default implementation to the deprecated method, you will not have to code an implementation here